external/fv3fit/tests/test_models.py

```diff
@@ -107,6 +108,26 @@ def test_DenseModel_clipped_inputs():
     xr.testing.assert_allclose(prediction_nan_filled, prediction_clipped, rtol=1e-3)
 
 
+def test_loaded_DenseModel_predicts_with_clipped_inputs(tmpdir):
```

Comment:
No need to change, but FYI in general there are a bunch of tests like this in `test_training.py`, you could get something like this in the future by parameterizing the dump-load-predict test there to use these special hyperparameters.